bin/_mocha

```diff
@@ -78,6 +78,14 @@ const exit = code => {
  */
 const list = str => str.split(/ *, */);
 
+/**
+ * Parse multiple flag.
+ */
+const collect = (val, memo) => {
```

Comment:
Could this just `return memo.concat(val)`, or be `Function.bind(Function.call, Array.prototype.concat)`?

Reply:
That it could. I took this straight from the commander.js page. I'm fine with it as is as it's pretty clear to any reader what the code is doing. Happy to change if people feel strongly otherwise.

Reply:
yeah, I agree this would be better:

```js
const collect = (val, memo) => memo.concat(val);
```